src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/locations/PlaceRecognizer.scala

```diff
@@ -1,62 +1,23 @@
 package com.microsoft.partnercatalyst.fortis.spark.transforms.locations
 
-import java.io.{IOError, IOException}
-
 import com.microsoft.partnercatalyst.fortis.spark.logging.Loggable
-import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.entities.EntityRecognizer
 import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.OpeNER
-import ixa.kaflib.Entity
-
-import scala.collection.JavaConversions._
-import scala.util.{Failure, Success, Try}
+import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.OpeNER.entityIsPlace
```

Comment:
nit: In this case, I think qualifying the name inline would make it more obvious to the reader that this helper is part of OpeNER.